mandatory/3-function-output.js

```diff
@@ -1,18 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
+  
 }
+// It generates a random number using Math.random() and it multiplies by 10
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
-
+// the concat() method is used to merge two or more arrays.
```

Comment:
:question: Is it just arrays that can use the `concat()` method? Did you come across any useful documentation when investigating?